redis.conf

```diff
@@ -199,6 +199,22 @@ tcp-keepalive 300
 #
 # tls-prefer-server-ciphers yes
 
+# By default, TLS session caching is enabled to allow faster and less expensive
+# reconnections by clients that support it. Use the following directive to disable
+# caching.
+#
+# tls-session-caching no
+
+# Change the default number of TLS sessions cached. A zero value sets the cache
+# to unlimited size. The default size is 20480.
+#
+# tls-session-cache-size 5000
```

Comment:
from my experience, most of these example lines contain the default.
in your case, ("Use the following directive to disable..") it makes sense to keep what you have.
so just for your consideration.

Reply:
Yes, earlier versions followed the same convention but it turned out to be more confusing to users and changed after getting some feedback.